report/2.Analysis/MotionTracking.tex

```diff
@@ -0,0 +1,46 @@
+\section{Object Tracking}
+
+To be able to track or know where the target is located, the device will require some sensors that can transmit information about the position of the desired target.
+One could easily imagine a couple of different sensors that could potentially gather information about the position of the target.
+For instance laser and sonar could both be used to gather some information about the position of a target\cite{Sensors}.
+However, since the system should be adaptable to a dynamic environment with potentially a lot of different objects in motion, one sensor was deemed the most obvious choice for this system; a camera.
+
+Using a camera is powerful, since images can be processed in order to determine what objects are present in the image.
+Once the location of the object is determined within the image, the direction of the target should be calculated.
+Assuming the camera is aimed in the same direction as the laser, the direction in which to aim, will simply be calculated based on the delta x and y coordinates of the target within the image.
+
+Unfortunately the camera does also come with a couple of cons.
+First of all, a normal camera will not be able to determine the distance to a target.
+Second of all, the camera will give a lot of excessive information, which will have to be processed in order to only keep the relevant information.
+As for the distance, the camera could potentially be used alongside a laser sensor used for the distance measuring\cite{Sensors}.
+
+\subsection{Image processing}
+Not only does the camera output too much information, but furthermore, the camera does not even provide the actual information needed.
+At least not directly that is.
+
+The image will have to be processed in order to figure out where the target is located within.
+Unfortunately it is no easy task to recognize objects in images.
```

Comment:
Denne linje og linjen under kan vist godt joines:
Unfortunately it is no easy task to recognize objects in images, and writing an algorithm that recognizes objects in images can be quite tedious.